### PR TITLE
Add Styling

### PR DESCRIPTION
**Pull request by devMahmoud**
In this milestone, I added styling to match Zeplin's design

### CODE REVIEW

File: src/App.css [hunk 1]
```diff
@@ -1,3 +1,13 @@
+* {
+  margin: 0;
+  padding: 0;
+}
+
 .App {
-  text-align: center;
+  padding: 6rem 2rem;
```
Comment (rustam-tolipov) [inline src/App.css]: - [x] Please kindly remove padding from `.App` and add `min-height: 100vh;` to make it full page app 👋🏼

File: src/components/Navbar/Navbar.css [hunk 2]
```diff
@@ -0,0 +1,53 @@
...
+  letter-spacing: 1.9px;
+  color: #121212;
+  display: flex;
+}
+
+.books-link {
+  width: 3.688rem;
+  height: 1rem;
+  text-decoration: none;
+}
```
Comment (rustam-tolipov) [inline src/components/Navbar/Navbar.css]: - [x] Please change nav link color to black and move them to the left as given in the design

**Screenshot**
image

File: src/components/Book/Book.css [hunk 3]
```diff
@@ -0,0 +1,68 @@
...
+  font-weight: 300;
+  letter-spacing: normal;
+  color: #4386bf;
+}
+
+.progress-container {
+  display: flex;
+  align-items: center;
+  margin-right: 1rem;
+}
```
Comment (rustam-tolipov) [inline src/components/Book/Book.css:52]: - [x] Please also increase progress percentage text font size

image

File: src/components/Book/Book.css [hunk 4]
```diff
@@ -0,0 +1,68 @@
...
+.chapter-num {
+  margin: 0.5rem 0;
+}
+
+.update-btn {
+  background-color: #4290fe;
+  color: #fff;
+  border: none;
+  padding: 0.25rem;
+}
```
Comment (rustam-tolipov) [inline src/components/Book/Book.css]: - [x] Let's also change buttons padding to `padding: 8px 16px;`

File: src/App.css [hunk 5]
```diff
@@ -1,3 +1,13 @@
...
+  background-color: #f5f6fa;
+}
+
```
Comment (rustam-tolipov) [inline src/App.css:10]: - [x] You should also add live between progress and current chapter

image

File: src/components/Navbar/Navbar.js [hunk 6]
```diff
@@ -1,11 +1,17 @@
...
-      <Link to="/">Books</Link>
-      <Link to="/categories">Categories</Link>
+    <span className="bookstore-title">
+      Bookstore CMS
+    </span>
+    <div className="nav-links">
+      <Link to="/" className="books-link">Books</Link>
+      <Link to="/categories" className="categories-link">Categories</Link>
     </div>
+    <img className="profile-icon" src={imgUrl} alt="" />
```
Comment (rustam-tolipov) [inline src/components/Navbar/Navbar.js:16]: - [x] For profile icons you can use this svg icon, simply download it and change the color 👍🏼

person

**Review by rustam-tolipov — CHANGES_REQUESTED** [review]
Hi @devMahmoud , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

## Highlights
_Your project is almost complete!_

emojibest_com_AnimatedSticker


## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
N/A

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by bimbolabuari — APPROVED** [review]
Hi @devMahmoud, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**